src/engraving/dom/playcounttext.h

```diff
@@ -19,19 +19,19 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-#ifndef MU_INSPECTOR_BARLINESETTINGSPROXYMODEL_H
-#define MU_INSPECTOR_BARLINESETTINGSPROXYMODEL_H
+#pragma once
 
-#include "models/abstractinspectorproxymodel.h"
+#include "textbase.h"
+#include "types/types.h"
 
-namespace mu::inspector {
-class BarlineSettingsProxyModel : public AbstractInspectorProxyModel
+namespace mu::engraving {
+class PlayCountText final : public TextBase
```

Comment:
I'm wondering if a new text type is actually needed, as it doesn't seem to add much to the base class. Would using the generic Text class be possible?

Reply:
Yeah, I ended up removing anything unique to it. I'll bin it!